cetlvast/suites/unittest/test_variable_length_array_bool.cpp

```diff
@@ -335,6 +335,38 @@ TYPED_TEST(VLABoolTests, TestBoolResizeOneBit)
     ASSERT_EQ(0, array[4]);
 }
 
+TYPED_TEST(VLABoolTestsVLAOnly, TestBoolExceedingMaxSizeMax)
```

Comment:
Can this be part of `VLABoolTests`? If not then what's the difference between vector and VLA here?

Reply:
No.  This test makes use of max_size_max which std::vector does not have.